explorer/interpreter/type_checker.cpp

```diff
@@ -2233,9 +2247,11 @@ auto TypeChecker::TypeCheckExp(Nonnull<Expression*> e,
           // This case handles the application of a parameterized class or
           // interface to a set of arguments, such as Point(i32) or
           // AddWith(i32).
-          const ParameterizedEntityName& param_name =
-              cast<TypeOfParameterizedEntityName>(call.function().static_type())
-                  .name();
+          const TypeOfParameterizedEntityName& entity =
+              cast<TypeOfParameterizedEntityName>(
+                  call.function().static_type());
+
+          const ParameterizedEntityName& param_name = entity.name();
```

Comment:
This feels like it's not necessary to change. I'm not seeing a particular style benefit to it, and you don't make use of `entity`. Can you please switch back so that we're not changing code when not necessary?